django_mongodb/managers.py

```diff
@@ -0,0 +1,44 @@
+from typing import TypeVar
+
+from django.db import models
+
+
+class MongoQuerySet(models.QuerySet):
+    """QuerySet which uses MongoDB as backend"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._prefer_search = False
+
+    def prefer_search(self, prefer_search=True):
```

Comment:
A small extension to `QuerySet`, which allows us mark a query as one, which is supposed to use the MongoDB search index.